botBackend/draft_logic.py

```diff
@@ -372,8 +372,8 @@ def invalid_pick(self, username: str, user_id: str, card_json: dict):
         if card_json["name"] in [cards for picks in self.picks.values() for cards in picks]:
             return "That card has already been chosen. Please try again."
 
-        if (self.format != "freeform" and card_json["legalities"][self.format] == "banned" or
-           card_json["legalities"][self.format] == "not_legal"):
+        if (self.format != "freeform" and (card_json["legalities"][self.format] == "banned" or
```

Comment:
minor logic error fix. Parenthesis help when you forget the order of and vs or and Boolean operations.